ci/docker/image-builder/README.md

```diff
@@ -0,0 +1,4 @@
+Docker image to build other images.
```

Comment:
I'm not sure that Readme better then comment inside dockerfile

Reply:
It depends on the scenario.

- Moved info about dependencies into buildscript.
- Added info about working with the project

#1341

Reply:
So add to Docker file `see details in README file`

Reply:
Got it. Will add in the next PR